Remove reference to deprecated method of specifying database (#171)
neo4j/docs-graphql: modules/ROOT/pages/driver-configuration.adoc

== Specifying the Neo4j database

> The database within a DBMS that should be used can be specified via the `sessionConfig` field in the `context` that all resolvers share.

Reviewer: ```suggestion
Specify the database to be used in your DBMS via the `database` field under `sessionConfig` in the `context` that all resolvers share.
```
double-check if the intended meaning is still captured
